submissions/nadsatt/friends-app/modules/filter-group.js

```diff
@@ -0,0 +1,93 @@
+import * as searchFiltersModule from './search-filter-items.js';
+import * as sortFiltersModule from './sort-filter-items.js';
+import { FilterList } from './filter-list.js';
+import { IconList } from './icon-list.js';
+
+export class FilterGroup {
+    constructor(userService, pageLinkList){
+        this.defineElement(userService, pageLinkList);
+        this.defineElementMethods();
+
+        return this.element;
```

Comment:
Can you give me link to the article or video which will describe pattern where you need to rewrite return value of constructor? Just want to see all picture :)